README.md

```diff
@@ -78,6 +78,95 @@ See also:
 
 ## Usage
 
+### Plugin
+
+**In order to further improve your production efficiency, we have also prepared plug-ins for you**
+
+- provider_template：[Android Studio/Intellij   Plugin](https://plugins.jetbrains.com/plugin/16821-flutter-provider)
+  - one-click generation of necessary folders, files, template codes, etc. for each page
+  - support Wrap Widget: Consumer, Selector, ChangeNotifirProvider
+  - Support input **provider** prompt to generate quick code snippets
+
+### Some usage
```

Comment:
I am not sure what this part adds over the existing documentation

Reply:
Hi，this is function of The plugin I wrote，It can easily generate widget refresh control

look

- pulgin
![image](https://user-images.githubusercontent.com/35656085/121760420-3c5c2180-cb5d-11eb-8792-da9b45064f6f.png)

- support Wrap Widget: Consumer, Selector, ChangeNotifirProvider
![image](https://user-images.githubusercontent.com/35656085/121760434-50078800-cb5d-11eb-9601-9685f851b0f7.png)

- Support input **provider** prompt to generate quick code snippets
![image](https://user-images.githubusercontent.com/35656085/121760437-57c72c80-cb5d-11eb-81c4-5294f1b7b55a.png)

Reply:
But what is the "Some usage" part about? It doesn't seem related to the plugin

Reply:
![image](https://user-images.githubusercontent.com/35656085/124581209-79c57d80-de83-11eb-92ae-e7c2ee79429e.png)
![image](https://user-images.githubusercontent.com/35656085/124581269-8518a900-de83-11eb-938a-e17e6109f372.png)


It can generate some code, you can choose to use it or not

Reply:
But what is the relationship with the doc and this?